lib/streamlit/elements/widgets/button_group.py

```diff
@@ -80,6 +81,8 @@
 # in base64 format and send it over the wire as an image.
 _SELECTED_STAR_ICON: Final = ":material/star_filled:"
 
+_FeedbackOptions2: TypeAlias = Literal["thumbs"]
+_FeedbackOptions5: TypeAlias = Literal["faces", "stars"]
 _FeedbackOptions: TypeAlias = Literal["thumbs", "faces", "stars"]
```

Comment:
Could make this a union of the 2 and 5 options.

Reply:
I tried this, because it also seemed like a logical idea to me, but it seems like mypy refuses to accept it. Possibly because the mypy version this project uses is not the latest version? (I've had a devil with the dependency system on this project 🤷‍♀️ so I didn't chase the possibility of upgrading). Or maybe I just didn't try the right variant.

Reply:
I have just tested it locally and `_FeedbackOptions: TypeAlias = _FeedbackOptions2 | _FeedbackOptions5` does not work for me either, but `_FeedbackOptions: TypeAlias = Union[_FeedbackOptions2, _FeedbackOptions5]` does